src/test/java/id/co/veritrans/mdk/v1/integration/vtdirect/BankTransferParamTest.java

```diff
@@ -0,0 +1,49 @@
+package id.co.veritrans.mdk.v1.integration.vtdirect;
+
+import id.co.veritrans.mdk.v1.gateway.model.CustomExpiry;
+import id.co.veritrans.mdk.v1.gateway.model.TransactionDetails;
+import id.co.veritrans.mdk.v1.gateway.model.vtdirect.BankTransferRequest;
+import id.co.veritrans.mdk.v1.gateway.model.vtdirect.paymentmethod.BankTransfer;
+import org.testng.annotations.Test;
+
+import id.co.veritrans.mdk.v1.helper.JsonUtil;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+import static org.testng.Assert.assertTrue;
+
+/**
+ * Testing for BankTransferRequest parameters
+ */
+public class BankTransferParamTest {
+
+    @Test
+    public void testBankTransfer() {
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss Z");
+        Date date = new Date();
+        try {
+            date = sdf.parse("2016-12-07 11:54:12 +0700");
```

Comment:
there should be not `catch` guard unless we want to assert the exception, so `Date date = sdf.parse("2016-12-07 11:54:12 +0700");` should be sufficient.